dhis-2/dhis-services/dhis-service-reporting/src/main/java/org/hisp/dhis/datasetreport/jdbc/AnalyticsDataSetReportStore.java

```diff
@@ -158,7 +158,7 @@ public Map<String, Object> getAggregatedSubTotals( DataSet dataSet, List<Period>
                 if ( filters != null )
                 {
                     params.addFilters(
-                        dataQueryService.getDimensionalObjects( filters, null, null, IdScheme.UID ) );
+                        dataQueryService.getDimensionalObjects( filters, null, null, null, IdScheme.UID ) );
```

Comment:
Would be good to somehow avoid adding many null arguments, makes the code more error prone and harder to read.

Reply:
please see above